examples/java-calculator/src/test/java/io/cucumber/examples/java/ParameterTypes.java

```diff
@@ -1,6 +1,6 @@
 package io.cucumber.examples.java;
 
-import io.cucumber.datatable.dependency.com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
What's the rationale behind changing from the shaded class?

Reply:
It'll be removed with https://github.com/cucumber/cucumber/pull/682.

This makes Jackson a complete drop in that can be managed by the user. Does however also require us to include it in the examples.